pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -1653,8 +1653,8 @@ def finalize_report():
             progress_logger.finish_dir(output_dir)
         print "\r\n\tHTML report written to %s" % report_preproc_filename
 
-    normalize = preproc_params["normalize"]
-    tsdiffana = preproc_params["tsdiffana"]
+    if "normalize" in preproc_params:
+        normalize = preproc_params["normalize"]
```

Comment:
This leaves the possibility solves the KeyError problem but opens another one: the normalize variable becomes undefined if preproc_params doesn't have the checked key.